easybuild/easyconfigs/g/gaussian/gaussian-09.e.01-AVX.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
```

Comment:
@akesandgren why? and if this is strictly required, we should support to specify an overriding `umask` value in the easyconfig file?